Mergin/projects_manager.py

```diff
@@ -241,7 +241,7 @@ def reset_local_changes(self, project_dir: str):
             QgsProject.instance().clear()
 
         try:
-            self.mc.reset_local_changes(project_dir)
+            self.mc.reset_local_changes(project_dir, files_to_reset)
```

Comment:
Let's add info box when we reset the changes

```suggestion
            self.mc.reset_local_changes(project_dir, files_to_reset)
            QMessageBox.info(... "File {filename} was successfully reset" || "Project local changes were successfully reset" )
```